scss/4-elements/_typography.scss

```diff
@@ -65,15 +50,55 @@ h4,
   }
 }
 
-p,
-.paragraph {
-  @include cads-paragraph();
-
+%p {
```

Comment:
:neckbeard:  Having `%p` but then `%paragraph--small` feels a touch confusing. Could this be `%cads-paragraph` and `%cads-paragraph-small`. Don't think we need the `--` modifier style as it's not really a modifier. 💯 to sharing the styles though.

Reply:
agree - I'll change all typography placeholders to `cads-`